### PR TITLE
Be/database update

### PR DESCRIPTION
**Pull request by K-MoraPais**

### CODE REVIEW

File: backend/anima-api/app/Http/Controllers/ControllerD.php [hunk 1]
```diff
@@ -8,101 +8,27 @@
...
             $newDonation = new Donation();
             $newDonation->idOlla = $request->input('idOlla');
-            $newDonation->userEmail = $request->input('userEmail');
-            $newDonation->donationType = $request->input('type');
-            $newDonation->date = date("Y/m/d");
+            $newDonation->email = $request->input('email');
+            $newDonation->tipoDonacion = $request->input('type');
+            $newDonation->fecha = date("Y/m/d");
             $newDonation->save();
             return "Donation stored successfully";
```
Comment (JhonAires-Anima) [inline backend/anima-api/app/Http/Controllers/ControllerD.php:29]: Maybe you should use `sendRequest()` and `sendError()` functions to have a more "professional" response.

File: backend/anima-api/app/Http/Controllers/ControllerO.php [hunk 2]
```diff
@@ -8,10 +8,10 @@
...
-    public function index()
+    public function getAll()
     {
-        $Ollas = Olla::where('state', '=', 1)
-            ->select('name', 'schedule', 'lat', 'long' ,'id', 'desc')
+        $Ollas = Olla::where('estado', '=', 1)
+            ->select('*')
             ->get();
 
         return $this->sendResponse($Ollas, '');
```
Comment (JhonAires-Anima) [inline backend/anima-api/app/Http/Controllers/ControllerO.php:17]: This sends automatically the response code?

Comment (JhonAires-Anima) [reply backend/anima-api/app/Http/Controllers/ControllerO.php:17]: And maybe in the empty string you can add a Succesful message

File: backend/anima-api/app/Models/User.php [hunk 3]
```diff
@@ -8,8 +8,8 @@
 class User extends Model
 {
     use HasFactory;
-    protected $connection='databasepe';
-    protected $table='users';
-    protected $primaryKey = "email";
+    protected $connection='olla_popular';
+    protected $table='usuario';
+    protected $primaryKey = "correo";
```
Comment (JhonAires-Anima) [inline backend/anima-api/app/Models/User.php:13]: Why you change between single quotes and double quotes?

File: backend/anima-api/app/Http/Controllers/ControllerU.php [hunk 4]
```diff
@@ -40,14 +40,15 @@ public function create()
      * @param  \Illuminate\Http\Request  $request
      * @return \Illuminate\Http\Response
      */
```
Comment (JhonAires-Anima) [inline backend/anima-api/app/Http/Controllers/ControllerU.php:42]: Remember to clear all unnecesary comments

**Review by JhonAires-Anima — CHANGES_REQUESTED** [review]

File: backend/anima-api/app/Http/Controllers/ControllerD.php (hunk 1 above)
Comment (K-MoraPais) [reply backend/anima-api/app/Http/Controllers/ControllerD.php]: Done

File: backend/anima-api/app/Http/Controllers/ControllerO.php (hunk 2 above)
Comment (K-MoraPais) [reply backend/anima-api/app/Http/Controllers/ControllerO.php]: I'm prioritizing login, registration and account activation tokens atm, so I'll resolve this later down the line.

File: backend/anima-api/app/Models/User.php (hunk 3 above)
Comment (K-MoraPais) [reply backend/anima-api/app/Models/User.php]: In some cases it is necessary, but not here. Resolved

File: backend/anima-api/app/Http/Controllers/ControllerU.php (hunk 4 above)
Comment (K-MoraPais) [reply backend/anima-api/app/Http/Controllers/ControllerU.php]: Laravel likes to add documentation stuff to autogenerated files, already deleted them. Resolved

File: backend/anima-api/app/Http/Controllers/ControllerO.php (hunk 2 above)
Comment (K-MoraPais) [reply backend/anima-api/app/Http/Controllers/ControllerO.php]: Done

File: backend/anima-api/.env [hunk 5]
```diff
@@ -28,13 +28,13 @@ REDIS_PASSWORD=null
...
-MAIL_ENCRYPTION=null
-MAIL_FROM_ADDRESS=null
-MAIL_FROM_NAME="${APP_NAME}"
+MAIL_HOST=smtp.gmail.com
+MAIL_PORT=465
+MAIL_USERNAME=cuentadescartablekevinm@gmail.com
+MAIL_PASSWORD=contraMuySegura9902
+MAIL_ENCRYPTION=ssl
+MAIL_FROM_ADDRESS=cuentadescartablekevinm@gmail.com
+MAIL_FROM_NAME=Token_Kevin_Mora
```
Comment (JhonAires-Anima) [inline backend/anima-api/.env:37]: You should change this to a name like "Ollas Populares"

File: backend/anima-api/routes/api.php [hunk 6]
```diff
@@ -13,9 +13,13 @@
...
-Route::apiResource('/olla', App\Http\Controllers\ControllerO::class);
-Route::apiResource('/user', App\Http\Controllers\ControllerU::class);
-Route::apiResource('/donation', App\Http\Controllers\ControllerD::class);
+Route::post('/activate', 'App\Http\Controllers\ControllerU@accountActivationHandler');
+Route::post('/ollas/save', 'App\Http\Controllers\ControllerO@createOlla');
+Route::get('/ollas', 'App\Http\Controllers\ControllerO@getAll');
+Route::post('/register', 'App\Http\Controllers\ControllerU@tempRegister');
+Route::post('/login', 'App\Http\Controllers\ControllerU@tempLogin');
+Route::get('/donations/{userEmail}', 'App\Http\Controllers\ControllerD@getDonationsFromUser');
+Route::post('/donations/save', 'App\Http\Controllers\ControllerD@saveDonation');
```
Comment (JhonAires-Anima) [inline backend/anima-api/routes/api.php:22]: You should add all the routes in the routes file.

**Review by JhonAires-Anima — CHANGES_REQUESTED** [review]

File: backend/anima-api/routes/api.php (hunk 6 above)
Comment (K-MoraPais) [reply backend/anima-api/routes/api.php:22]: I only add those that are finished, if they are not in the Routes.txt file, then they are not ready.

File: backend/anima-api/.env (hunk 5 above)
Comment (K-MoraPais) [reply backend/anima-api/.env:37]: Sure, i'll have it done tomorrow.

**Comment by braimor** [comment]
Remember to manage the "Estado" field with a default value of "Con necesidad".

File: backend/anima-api/.env (hunk 5 above)
Comment (K-MoraPais) [reply backend/anima-api/.env]: Done

**Review by JhonAires-Anima — APPROVED** [review]

**Pull request merged**